scripts/deploy-cdn-rsync.sh

```
@@ -0,0 +1,126 @@
+    echo -e "${YELLOW}🔐 Setting up SSH key from GitHub secrets...${NC}"
+
+    # Create SSH directory and set permissions
+    mkdir -p ~/.ssh
+    chmod 700 ~/.ssh
+
+    # Write the private key to a file
+    echo "$SSH_PRIVATE_KEY" > ~/.ssh/id_rsa
+    chmod 600 ~/.ssh/id_rsa
+
+    # Add the server to known_hosts to avoid host key verification
+    ssh-keyscan -p $SERVER_PORT -H $SERVER_HOST >> ~/.ssh/known_hosts 2>/dev/null
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)